c10/core/impl/LocalDispatchKeySet.cpp

```diff
@@ -101,4 +101,8 @@ void tls_set_dispatch_key_included(DispatchKey x, bool desired_state) {
   }
 }
 
+bool tls_is_dispatch_keyset_excluded(DispatchKeySet ks) {
```

Comment:
maybe less ambiguous with `is_all_dispatch_keyset_excluded` (the ambiguity is because `any` qualifier is also possible here)